버드콩_윤세빈/1주차_미션/미션1/미션1.css

```
@@ -0,0 +1,48 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+}
+
+.navbar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 30px 20px;
+    background-color: white;
+    /* box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1); */
+}  
+
+
+@media screen and (max-width: 600px) {
+    .logo-text {
+        display: none;  /* 화면 크기가 600px 이하일 때 글씨 숨기기 */
+    }
```

화면 크기가 줄었을 때 여전히 해가 잘려보이는데

```suggestion
    .nav-links {
        display: flex;
        justify-content: space-evenly;
        flex: 1;
        gap: 0px !important;
    }
```

를 미디어 쿼리에 추가하고 해 이모지를 .nav-links 안으로 넣으면 쉽게 해결될 것 같습니다. 자세한건 본인이 해보는 걸루... 하띵....